### DISCUSSION TITLE
Muligheter for værmelding

### DISCUSSION THREAD

**Discussion by SebastianHellum** [entur/tavla#307] [Ideas]
Ofte når man skal ut for å ta bussen kan det være greit å sjekke værmeldingen også. Kunne det vært en ide å hatt et lite værsymbol og antall grader ved siden av klokken på tavla?

**Comment by draperunner** [comment]
Hei! Me har tenkt litt på å ha ei slik værmelding i Tavla, ja. Me må få spikra designet me ønsker og så finne tid til å implementere det. Så over nyttår ein gong, kanskje.

**Comment by forteller** [comment]
Kult. Håper det blir i en manipulerbar boks.

**Comment by draperunner** [comment]
Kan melde om at me ønsker å få til værmelding i løpet av sommaren. ☀️

**Comment by magnusrand** [comment]
Jeg kan meddele at værmelding nå endelig er lagt til i Entur-Tavla! 🥳 🌤️

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
